src/main/java/com/czertainly/core/api/cmp/CmpController.java

```diff
@@ -0,0 +1,89 @@
+package com.czertainly.core.api.cmp;
+
+import com.czertainly.api.model.core.acme.ProblemDocument;
+import com.czertainly.core.api.cmp.error.CmpBaseException;
+import io.swagger.v3.oas.annotations.ExternalDocumentation;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping({"/v1/protocols/cmp/{cmpProfileName}"})
+@Tag(
+        name = "CMP operations",
+        description = "Interfaces used by CMP clients to request CMP related operations. CMP Profile defines the behaviour for the specific CMP configuration. When the CMP Profile contains default RA Profile, it can be used by the CMP clients to request operations on their specific URL."
+)
+@ApiResponses({@ApiResponse(
+        responseCode = "400",
+        description = "Bad Request",
+        content = {@Content(
+                mediaType = "application/problem+json",
+                schema = @Schema(
+                        implementation = ProblemDocument.class
+                )
+        )}
+), @ApiResponse(
+        responseCode = "401",
+        description = "Unauthorized",
+        content = {@Content(
+                mediaType = "application/problem+json",
+                schema = @Schema(
+                        implementation = ProblemDocument.class
+                )
+        )}
+), @ApiResponse(
+        responseCode = "403",
+        description = "Forbidden",
+        content = {@Content(
+                mediaType = "application/problem+json",
+                schema = @Schema(
+                        implementation = ProblemDocument.class
+                )
+        )}
+), @ApiResponse(
+        responseCode = "500",
+        description = "Internal Server Error",
+        content = {@Content}
+)})
+public interface CmpController {

```

Comment:
This `interface` should be moved to https://github.com/3KeyCompany/CZERTAINLY-Interfaces/pull/372.

Reply:
DONE, api/error moved into Interfaces.